src/table/utils/find-visible-rows.ts

```diff
@@ -36,6 +38,53 @@ const findVisibleRows = (rootElement: HTMLElement, totalsPosition: TotalsPositio
   const visibleRowStartIndex = findStartIndex(dataRows, yMin, yMax);
   const visibleRowEndIndex = findEndIndex(dataRows, yMin, yMax);
   return { visibleRowStartIndex, visibleRowEndIndex };
+}
+
+const getFristCellOfRow = (rowIndex: number, cells?: NodeListOf<Element>) => {
+  if (!cells) return undefined;
+  for (let i = 0; i < cells.length; i++) {
+    const strIdx = cells[i].getAttribute('rowindex');
+    if (strIdx && parseInt(strIdx, 10) === rowIndex) return cells[i];
+  }
+  return undefined;
+};
```

Comment:
Probably will solve the  eslint problem of having `return undefined` twice...
```suggestion
  if (cells) {
    for (let i = 0; i < cells.length; i++) {
    const strIdx = cells[i].getAttribute('rowindex');
    if (strIdx && parseInt(strIdx, 10) === rowIndex) return cells[i];
     }
  } else return undefined;
};
```